pkg/daemon/ceph/osd/volume.go

```diff
@@ -66,6 +66,10 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 	if err != nil {
 		return nil, fmt.Errorf("failed to generate osd keyring. %+v", err)
 	}
+	// Update LVM configuration file
+	if err := updateLVMConfig(context, true); err != nil {
```

Comment:
The second argument is `true` every time. Could be a problem with `osd on disks`.  We can use `a.pvcBacked` here.

Reply:
Other than this, LGTM